Annex60/Fluid/Actuators/BaseClasses/PartialTwoWayValve.mo

```diff
@@ -29,7 +32,10 @@ protected
 initial equation
   assert(dpFixed_nominal > -Modelica.Constants.eps, "Require dpFixed_nominal >= 0. Received dpFixed_nominal = "
         + String(dpFixed_nominal) + " Pa.");
-
+equation
+  if assertPhiPos then
+    assert(phi>0, "Negative valve control signals are not allowed, but found phi=" + String(phi));
```

Comment:
Zero, which is not negative, is also not allowed. Hence, a better comment is "Valve control signal needs to be bigger than zero, received "